### DISCUSSION TITLE
2022/02/13/Java%E8%BF%9E%E6%8E%A5Mysql%E5%AD%97%E7%AC%A6%E9%9B%86%E8%AE%BE%E7%BD%AE%E8%BF%87%E7%A8%8B%E8%AF%A6%E8%A7%A3/

### DISCUSSION THREAD

**Discussion by giscus** [skitii/skitii#6] [General]
# 2022/02/13/Java%E8%BF%9E%E6%8E%A5Mysql%E5%AD%97%E7%AC%A6%E9%9B%86%E8%AE%BE%E7%BD%AE%E8%BF%87%E7%A8%8B%E8%AF%A6%E8%A7%A3/

utf8mb4设置无效引发的java-mysql字符集设置探索

https://skitii.vercel.app/2022/02/13/Java%E8%BF%9E%E6%8E%A5Mysql%E5%AD%97%E7%AC%A6%E9%9B%86%E8%AE%BE%E7%BD%AE%E8%BF%87%E7%A8%8B%E8%AF%A6%E8%A7%A3/

**Comment by AbnerHuang2** [comment]
为什么这个评论加载这么慢。
